packages/flame/lib/src/anchor.dart

```diff
@@ -54,6 +54,7 @@ class Anchor {
     if (this == otherAnchor) {
       return position;
     } else {
+      // TODO(Any): This should not be creating new Vector2 objects.
```

Comment:
Can we open an issue to track this TODO?

Reply:
Good idea: https://github.com/flame-engine/flame/issues/2543